src/projects/EnsureThat/Enforcers/StringArg.cs

```diff
@@ -37,6 +37,13 @@ public string IsNotNullOrEmpty([ValidatedNotNull]string value, [InvokerParameter
             return value;
         }
 
+        public string IsEmptyOrWhitespace(string value, [InvokerParameterName] string paramName = null, OptsFn optsFn = null)
```

Comment:
I believe you may have meant `Is**Not**EmptyOrWhitespace`

Reply:
Man, this makes me look stupid. Don't drink and code.